<ui version="4.0" >
 <class>OverwriteDialog</class>
 <widget class="QDialog" name="OverwriteDialog" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>389</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>550</width>
    <height>286</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>350</y>
     <width>531</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" >
    <item>
     <widget class="KPushButton" name="cancelButton" >
      <property name="text" >
       <string>&amp;Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KPushButton" name="renameButton" >
      <property name="text" >
       <string>&amp;Rename</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="KPushButton" name="yesButton" >
      <property name="text" >
       <string>&amp;Yes</string>
      </property>
      <property name="flat" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KPushButton" name="noButton" >
      <property name="text" >
       <string>&amp;No</string>
      </property>
      <property name="checkable" >
       <bool>false</bool>
      </property>
      <property name="checked" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>535</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <item row="2" column="0" >
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QLabel" name="destIconLabel" >
        <property name="minimumSize" >
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolTip" >
         <string/>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="scaledContents" >
         <bool>false</bool>
        </property>
        <property name="openExternalLinks" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTableWidget" name="destDetails" >
        <property name="minimumSize" >
         <size>
          <width>439</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>439</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="verticalScrollBarPolicy" >
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="autoScroll" >
         <bool>true</bool>
        </property>
        <property name="autoScrollMargin" >
         <number>5</number>
        </property>
        <property name="editTriggers" >
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="horizontalScrollMode" >
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="showGrid" >
         <bool>false</bool>
        </property>
        <property name="gridStyle" >
         <enum>Qt::DashLine</enum>
        </property>
        <property name="cornerButtonEnabled" >
         <bool>true</bool>
        </property>
        <property name="rowCount" >
         <number>4</number>
        </property>
        <property name="columnCount" >
         <number>2</number>
        </property>
        <row/>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0" >
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <widget class="QLabel" name="sourceIconLabel" >
        <property name="minimumSize" >
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="scaledContents" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTableWidget" name="sourceDetails" >
        <property name="minimumSize" >
         <size>
          <width>439</width>
          <height>95</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>439</width>
          <height>95</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="verticalScrollBarPolicy" >
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="autoScroll" >
         <bool>true</bool>
        </property>
        <property name="autoScrollMargin" >
         <number>5</number>
        </property>
        <property name="editTriggers" >
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="horizontalScrollMode" >
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="showGrid" >
         <bool>false</bool>
        </property>
        <property name="gridStyle" >
         <enum>Qt::NoPen</enum>
        </property>
        <property name="rowCount" >
         <number>3</number>
        </property>
        <property name="columnCount" >
         <number>2</number>
        </property>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0" >
     <widget class="QLabel" name="label_3" >
      <property name="font" >
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="lineWidth" >
       <number>1</number>
      </property>
      <property name="midLineWidth" >
       <number>1</number>
      </property>
      <property name="text" >
       <string>Would you like to replace it with this one?</string>
      </property>
      <property name="scaledContents" >
       <bool>false</bool>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
      <property name="indent" >
       <number>0</number>
      </property>
      <property name="textInteractionFlags" >
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <spacer name="verticalSpacer" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0" >
     <spacer name="verticalSpacer_2" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="font" >
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>The following file already exists</string>
      </property>
      <property name="indent" >
       <number>0</number>
      </property>
      <property name="textInteractionFlags" >
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0" >
     <spacer name="verticalSpacer_3" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>320</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text" >
    <string>Hold down the Yes or No button to apply to all files</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
